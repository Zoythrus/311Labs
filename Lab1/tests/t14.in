radix
100 9
943925373
519780090
548539079
584033542
949700339
993939058
547013621
980222611
423521081
493709225
320128407
789989905
962454214
209364868
204606417
361659495
692349759
948656792
611851126
925393428
636968411
235070033
287788898
392270006
434023959
616252654
540655270
863498872
790356915
985600187
436423354
336108486
840572918
382804481
981249847
773679817
839272700
715832623
310017551
451729252
284244290
989901694
774632100
529283796
710167943
481928803
209622114
156384801
460769367
486563892
972869692
596875996
793915381
631277708
253786623
429514399
655487946
158774800
283763766
773574204
375591141
195823001
833000549
439001974
294331083
625941742
514638622
676328153
244276418
376751696
944857760
548058271
290572424
978215372
130354946
607967944
806761351
572837366
468149333
277512955
641216841
317217217
312005876
356666445
240296973
630927179
214294022
638065337
944477096
739023387
609746070
505747989
581155193
956916150
592772125
268311130
329117880
764773176
169555076
