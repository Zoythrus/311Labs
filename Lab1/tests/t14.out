130354946
156384801
158774800
169555076
169555076
195823001
204606417
209364868
209622114
214294022
235070033
240296973
244276418
253786623
268311130
277512955
283763766
284244290
287788898
290572424
294331083
310017551
312005876
317217217
320128407
329117880
336108486
356666445
361659495
375591141
376751696
382804481
392270006
423521081
429514399
434023959
436423354
439001974
451729252
460769367
468149333
481928803
486563892
493709225
505747989
514638622
519780090
529283796
540655270
547013621
548058271
548539079
572837366
581155193
584033542
592772125
596875996
607967944
609746070
611851126
616252654
625941742
630927179
631277708
636968411
638065337
641216841
655487946
676328153
692349759
710167943
715832623
739023387
764773176
773574204
773679817
774632100
789989905
790356915
793915381
806761351
833000549
839272700
840572918
863498872
925393428
943925373
944477096
944857760
948656792
949700339
956916150
962454214
972869692
978215372
980222611
981249847
985600187
989901694
993939058
